README.md

```diff
@@ -6,36 +6,80 @@ You can extract the actual TOTP seed and use it with apps like [KeePassXC](https
 
 ## Prequisites
 
-For this to work, you need to extract 2 things - [SSAID](https://developer.android.com/reference/android/provider/Settings.Secure#ANDROID_ID) and encrypted seed.
+For this to work, you need to extract 3 things - [SSAID](https://developer.android.com/reference/android/provider/Settings.Secure#ANDROID_ID), UUID and encrypted seed.
 This needs root access on your Android device.
 
 ### SSAID
 
-Run this command in a rooted shell:
+#### Android 8.0+
+For Android 8.0+ SSAID is unique for every installed app, to get value for FTM run this command in a rooted shell:
 ```
 # grep com.fortinet /data/system/users/0/settings_ssaid.xml                                                                 
 ```
 
 Output should look like this:
 ```xml
-<setting id="32" name="10309" value="12dddfc4a3b45678" package="com.fortinet.android.ftm" defaultValue="12dddfc4a3b45678" defaultSysSet="false" tag="null" />
+<setting id="32" name="10309" value="eefd7d4837294e94" package="com.fortinet.android.ftm" defaultValue="eefd7d4837294e94" defaultSysSet="false" tag="null" />
```

Comment:
this was a bogus value, i hope you didn't replace it (and the other two) with your actual ones

Reply:
They are fake values from the article.